### DISCUSSION TITLE
MQTT.js dependencies

### DISCUSSION THREAD

**Discussion by es4m0n** [mqttjs/MQTT.js#1858] [Q&A] [answered]
Hello everyone,

i m not a experienced develop and have a question to mqtt.js dependecies. I want to hoste a Website on my Raspberry Pi with a NGINX Server which should use a local mqtt.js installation for MQTT over Websockets because i cant use a CDN link due to the lack of Internet access.

My questions is, can i install just mqtt.js or do i have to install Node.js first. 

Thank you advance!

**Comment by robertsLando** [comment, marked as answer]
You can simply download a release zip and just import on your html file one of the bundles available in `dist` folder https://github.com/mqttjs/MQTT.js/releases/tag/v5.5.5

Of course you should setup your nginx in order it to be able to serve that file from your local filesystem
